static/js/publisher/release/components/releasesConfirmDetails/globalRow.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { InteractiveProgressiveBar } from "../progressiveBar";
+
+const GlobalRow = ({
+  useGlobal,
+  toggleGlobal,
+  globalPercentage,
+  updatePercentage
+}) => {
+  return (
+    <div className="p-releases-confirm__details-global row u-vertically-center">
+      <div className="col-4">
+        <label>
+          <span>
+            <span className="p-tooltip--btm-center">
+              <span className="p-help">
+                Use the same progressive release percentage
+              </span>
+              <span className="p-tooltip__message">
+                For new progressive releases
+              </span>
+            </span>
+          </span>
+          <span className="p-releases-confirm__details-switch">
+            <input
+              type="checkbox"
+              className="p-switch"
+              checked={useGlobal}
+              onChange={toggleGlobal}
+            />
+            <div className="p-switch__slider" />
+          </span>
+        </label>
+      </div>
+      {useGlobal && (
+        <div className="col-4 p-release-details-row__progress">
+          <InteractiveProgressiveBar
+            percentage={globalPercentage}
+            targetPercentage={globalPercentage}
+            minPercentage={1}
+            singleDirection={0}
+            onChange={updatePercentage}
+          />
+          <span>
```

Comment:
Again - why additional span?

Reply:
This one is, because of `display: flex` of the parent, without it the tooltip doesn't align with the text.